command/flags/http.go

```diff
@@ -174,3 +179,9 @@ func (f *HTTPFlags) AddPartitionFlag(fs *flag.FlagSet) {
 			"from the request's ACL token, or will default to the `default` admin partition. "+
 			"Admin Partitions are a Consul Enterprise feature.")
 }
+
+func (f *HTTPFlags) AddPeerName() *flag.FlagSet {
+	fs := flag.NewFlagSet("", flag.ContinueOnError)
+	fs.Var(&f.peer, "peer", "Specifies the name of peer to query. By default, it is `local`.")
+	return fs
+}
```

Comment:
Why is this added specifically for the resource API? I'm wondering if we should hold on for peering as it's not clear how we will implement the api? 🤔

Reply:
@dhiaayachi 
I'm not sure how the `peer` flag is used by the backend, I added this flag here based on the AC of the ticket, it basically requires the `peer` information, that's why I added it here.

Please let me know if the information here is redundant or not.

Reply:
@dhiaayachi The `peer` field is part of the resource gRPC API hence it's included in both the HTTP side and the CLI side. 
The pending discussion/work is about whether peer should be moved outside the tenancy block but I am not sure what you mean by "we are not clear how we will implement the api" can you elaborate?

Reply:
I agree I think supporting both make sense.